migrator-tool/src/main/scala/za/co/absa/spline/migrator/MigratorCLI.scala

```diff
@@ -35,6 +35,11 @@ object MigratorCLI extends App {
 }
 
 class MigratorCLI(migratorTool: MigratorTool) {
+
+  private val logLevels = Seq("ALL", "TRACE", "DEBUG", "INFO", "WARN", "ERROR", "OFF")
+  private val logLevelsString = logLevels.map(x => s"`$x`").reduce((x, y) => s"$x, $y")
+  private val logLevelInvalidMsg = s"<log-level> should be one of: $logLevelsString"
```

Comment:
I would rather inline this constant, just like we inline all the other parser messages. The reason is that it only has a single usage and it's specific to one single parser rule. The same holds for the rest of these constants. Alternatively if you want to reuse, say `logLevels` you could move those vals to the parser rule block where they are used. E.g. 
```scala
{
  val a = ...
  val b = ...
  (opt[...](...))
}
```